### ISSUE TITLE
Category Request: Ens Bookclub (All words containing ENS)

### ISSUE THREAD

**Issue by yabyumdoteth** [Zimtente/ens-collections#61]
name: "Ens Bookclub"
slug: "ens-bookclub"
description: All words containing ENS
twitter: 
website:
chat:
![ensbookclub png](https://user-images.githubusercontent.com/76005714/176392249-504f70e9-f4a8-49dd-a9d5-506f71700ccb.png)

logo:"ensbookclub.png"
[ens-name-tokenIds-1656449140274.csv](https://github.com/Zimtente/ens-collections/files/9008993/ens-name-tokenIds-1656449140274.csv)
]

This collection is made of only words that contains the letters/words ENS. There's around 940 words thus making this collection unique in scarcity. Let's make ENS bookclub a thing.

**Comment by tunnckoCore** [comment]
"bookclub"?! definitely should be called something else. When you see "bookclub" you don't think of such thing.

**Comment by yabyumdoteth** [comment]
> "bookclub"?! definitely should be called something else. When you see "bookclub" you don't think of such thing.

You're right, i put it there as an example. If you have a better name for this theme let me know and i'll update.
